layouts/partials/footer_suffix.html

```diff
@@ -14,4 +14,15 @@
     type: "filteredEvents"
   });
 })(jQuery, document);
+
+jQuery(document).ready(function($){ 
```

Comment:
Not sure why you need to do this?

Reply:
This is for dropdown mobile sidebar, I created 2, one is dropdown and the other is collapsible, and I will defer it to the team to choose one, and I will drop the other one.

Reply:
Is there a place where both options are being displayed so that we can make a decision?

Reply:
Yes, on IOT (all pages where sidebar exists, my mobile sidebar goes the same when it has a sidebar or not) :
https://deploy-preview-539--eclipsefdn-iot.netlify.app/community/resources/
https://deploy-preview-539--eclipsefdn-iot.netlify.app/community/
https://deploy-preview-539--eclipsefdn-iot.netlify.app/adopters/how-to-be-listed-as-an-adopter/
https://deploy-preview-539--eclipsefdn-iot.netlify.app/projects/getting-started/

on Hugo theme:
https://deploy-preview-174--eclipsefdn-hugo-solstice-theme.netlify.app/third-parent
https://deploy-preview-174--eclipsefdn-hugo-solstice-theme.netlify.app/membership/
https://deploy-preview-174--eclipsefdn-hugo-solstice-theme.netlify.app/


Thanks!

Reply:
I like the collapsable option but I think it would need a down arrow or something. It took me a minute to understand that this was the menu. Or maybe the hamburger icon. Something that would tell the users that this is a menu or that there is more options available to click.

Reply:
I decide to use a hamburger like the menu.

Reply:
But for resources, I'm using a pre that is the same as the normal sidebar